packages/example-react/stories/Multiple.md

```diff
@@ -0,0 +1,25 @@
+## Title
+
+Some description
+
+```js
+// const Button = require('../components/Button');
+
+<button>Hello World</button>
+```
+
+```js
+// const Button = require('../components/Button');
```

Comment:
Right now don't know how correctly import components.
It should work in context of md file dir.